tests/unittests/test_backoff.py

```diff
@@ -0,0 +1,72 @@
+import unittest
+import FuelSDK
+import socket
+import time
+import singer
+from unittest import mock
+from tap_exacttarget.endpoints.emails import EmailDataAccessObject
+
+class Mockresponse:
+    def __init__(self, status, json):
+        self.status = status
+        self.results = json
+        self.more_results = False
+
+def get_response(status, json={}):
+    return Mockresponse(status, json)
+
+@mock.patch("FuelSDK.ET_Email.get")
+@mock.patch("time.sleep")
+class TestConnectionResetError(unittest.TestCase):
```

Comment:
Add tests for all fields. Better to have more unit tests

Reply:
Added test cases for all fields.